docs/src/use/migrate-to-9.0.0.md

```diff
@@ -130,3 +132,23 @@ ESLint v9.0.0 removes the deprecated `sourceCode.getComments()` method.
 **To address:** Replace with `sourceCode.getCommentsBefore()`, `sourceCode.getCommentsAfter()`, or `sourceCode.getCommentsInside()`.
 
 **Related Issues(s):** [#14744](https://github.com/eslint/eslint/issues/14744)
+
+## <a name="drop-function-style-rules"></a> Function-style rules are no longer supported
+
+ESLint v9.0.0 drops support for function-style rules. Function-style rules are rules created by exporting a function from a file rather than exporting an object with a `create()` method. This rule format was deprecated in 2016.
```

Comment:
Defining one rule per file is definitely best practice, but I have seen the occasional plugin that defines all their rules in a single file. To be agnostic about how a plugin's files are structured, we could omit "from a file", although this isn't too big of a deal.

```suggestion
ESLint v9.0.0 drops support for function-style rules. Function-style rules are rules created by exporting a function rather than an object with a `create()` method. This rule format was deprecated in 2016.
```

Reply:
Agreed, makes sense to not mention files.